pymemcache/client/hash.py

```diff
@@ -121,22 +121,29 @@ def remove_server(self, server, port):
         key = '%s:%s' % (server, port)
         self.hasher.remove_node(key)
 
+    def _retry_dead(self):
+        current_time = time.time()
+        ldc = self._last_dead_check_time
+        # we have dead clients and we have reached the
+        # timeout retry
```

Comment:
Note that the guarantee that we have dead clients is now external to this function.

Reply:
That's one way of viewing it. The other is that this function works regardless of whether we have dead clients or not - meaning the check for empty clients is simply an optimisation.

I'll update the comment accordingly.